package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "freedom",
   "description": "Embracing a distributed web",
-  "version": "0.6.25",
+  "version": "0.6.26",
```

Comment:
It's best not to bump version numbers directly like this, but instead use the `grunt release` task (this applies to the other `freedom-for-*` as well, which should have similar tasks).